.github/workflows/step_build-wheel.yaml

```diff
@@ -43,11 +33,9 @@ jobs:
         with:
           platforms: all
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.16
+        uses: pypa/cibuildwheel@v2.17
         env:
           CIBW_BUILD: ${{ inputs.cibw_build }}
-          CIBW_ARCHS_LINUX: ${{ inputs.cibw_archs_linux }}
-          CIBW_ARCHS_MACOS: ${{ inputs.cibw_archs_macos }}
```

Comment:
Does it mean we do not need to manually specify `CIBW_ARCHS_LINUX` and `CIBW_ARCHS_MACOS` anymore?

Reply:
Yes, we are only using native builders anyway and with `macOS-14` there is M1-arm native builders

Reply:
Oh, wait the linux arch is not picked up :/